pylint/checkers/refactoring/refactoring_checker.py

```diff
@@ -1882,6 +1882,7 @@ def _check_unnecessary_dict_index_lookup(
                     if isinstance(value, nodes.Name):
```

Comment:
The comment above is outdated but I can't suggest on it directly. 

```diff
- # Case where .items is assigned to k,v (i.e., for k, v in d.items())
+ # Case where .items is assigned (i.e., for k, v in d.items() or for k, in d.items())
```

Reply:
Isn't the comment still correct? Anything that's not assignment to a 2-tuple is still filtered out, i.e. I read this comment as applying to everything after the `continue`. We're just writing a better `continue` condition :-)

Reply:
Hmm, ok I did not see it like that but you're right. It will still be hard to tell what `` len(node.target.elts) < 2`` is doing without the context of this MR though. Maybe we could add a comment or create a named intermediary boolean ?